tox.ini

```diff
@@ -28,7 +28,11 @@ commands =
   /bin/bash -c 'podman pull -q {[base]default_ee} || docker pull -q {[base]default_ee}'
   /bin/bash -c 'podman pull -q {[base]small_test_ee} || docker pull -q {[base]small_test_ee}'
   # most coverage options are kept in pyproject.toml
-  /bin/bash -c 'py.test -vvvv -n `python -c "import multiprocessing; import os; print(int(os.environ.get(\"XDIST_CPU\") or multiprocessing.cpu_count()))"` --dist=loadfile --maxfail=15 --durations=100 --cov --cov-report=xml:{toxworkdir}/coverage-{envname}.xml --showlocals {posargs}'
+  # if you want to limit multiprocessing locally, define PYTEST_ADDOPTS="-n0 --dist no"
+  pytest {posargs:-vvvv \
+    -n auto --dist=loadfile --maxfail=15 \
+    --durations=100 --cov --cov-report=xml:{toxworkdir}/coverage-{envname}.xml --showlocals \
+    {env:PYTEST_ADDOPTS}}
```

Comment:
This changes the pytest behavior to be different from what a regular pytest user would expect.
The official documentation shows how this environment variable is supposed to be used, which is exactly opposite to what this does — https://docs.pytest.org/en/6.2.x/reference.html#envvar-PYTEST_ADDOPTS.

Reply:
Last two comments lack one essential part: suggestion an acceptable alternative for addressing the goal of the this pull request. How can we achieve the stated goals?

IMHO, the re-use of that option is not really a big issue. Obviously that we could use another environment variable name to avoid that, bug do we really need another one?

Reply:
Okay, use another variable but don't change the semantics of the well-known one.